### DISCUSSION TITLE
如何使用nebo的笔和橡皮擦切换

### DISCUSSION THREAD

**Discussion by ghost** [eiyooooo/MateBook-E-Pen#5] [General]
冒昧问一下

好像怎么试都不支持捕获控件组合

**Comment by eiyooooo** [comment]
> 冒昧问一下
> 
> 好像怎么试都不支持捕获控件组合

UWP应用尝试用鼠标捕获
我试了试，nebo的可视化控件层级好像会随操作改变，没办法准确捕获并保存
